### DISCUSSION TITLE
Addition of Control Groups

### DISCUSSION THREAD

**Discussion by Aklugua** [Rem0o/FanControl.Releases#824] [Ideas]
Here is the idea (haven't seen it in other threads so i'm creating one),

A user could add a **Control Group** (that can be renamed of course), and then add existing **Controls** to it (in the same way that we currently add existing sensors to custom ones) and apply the selected curve, so if we take the example below :
![Fan Control - Controls](https://user-images.githubusercontent.com/31726770/170367879-5005c280-ae45-4e58-9241-9ab2c48474a6.png)

You could create two **Control Groups** :
1. The first one would be a combination of the 2 CPU fan **Controls** using the _CPU FC/Temp_ curve
2. The second one would be a combination of the 4 case fan **Controls** using the _FC/Temp_ curve

From what i see, a **Control Group** would either be a _Special_ **Control** that would be added by the user, or a new category altogether on top of current **Controls**.

Could it be feasable ?

**Comment by Rem0o** [comment]
Feasible yes, but what does that added complexity adds to the table?

**Reply by Aklugua** [reply]
Mainly to reduce visual clutter in case of there being a lot of controls (you could then hide the similar controls and only keep the few groups that are being used, creating a form of _hierarchy_ between **Controls**).

Although this isn't an issue per say, that would probably be a nice feature for people using this application with small screens (or people that enjoy keeping things minimalistic like i do).

But i concur with the fact that implementing this isn't critical and is purely visual.

**Comment by ghost** [comment]
Hi Rem0o.
Awesome software.
Just a few recommendations.
would be good to have a check box toggle that would make the checked "hidden" fans that are grayed out, dissapear to clean up the dashboard.

I would also like to be able to add fans to a fan curve profile.
Ive an open bench and its got basically two sides / sets.
CPU which is an AIO "pump and its 2 fans"
and a 1080 which has its built in 3 fans plus two on its rear side cooling the backplate.

the software "or i have not figured it out yet" does not see the aio pump. "asus ryujin 2 240"
I would like to group to GPU the 3 inbuilt and 2 back fans so they all work in sync

Im also playing with the idea of adding for oc testing some VRM fans which would then behave the same as the gpu setup.
and in that instance id also like to add another group to the CPU curve so that all CPU related fans work off a single curve.

R
Dan

**Reply by Rem0o** [reply]
> would be good to have a check box toggle that would make the checked "hidden" fans that are grayed out, dissapear to clean up the dashboard.

There is in the left hamburger menu

> I would like to group to GPU the 3 inbuilt and 2 back fans so they all work in sync

Use a sync fan curve.
